src/tools/fuzzing/fuzzing.cpp

```diff
@@ -4413,6 +4438,55 @@ Expression* TranslateToFuzzReader::makeBulkMemory(Type type) {
   WASM_UNREACHABLE("invalid value");
 }
 
+Expression* TranslateToFuzzReader::makeTableGet(Type type) {
+  // Emit a get from the funcref table (which always exists) or the exnref one
+  // (which might not, if EH is disabled).
+  auto makeTableGet = [&](Name tableName) {
+    auto* table = wasm.getTable(tableName);
+    // Usually emit in-bounds gets, to avoid trapping, but rarely allow
+    // anything.
+    Expression* index;
+    if (allowOOB && oneIn(10)) {
+      index = make(table->addressType);
+    } else {
+      index = builder.makeConst(
+        Literal::makeFromInt32(table->initial, table->addressType));
```

Comment:
Do we only get from the initial index? (The same for `table.set`)

Reply:
Good catch! This should be `upTo(table->initial)`. Fixed.

Reply:
So we don't support `table.grow` in the fuzzer and the size is the same as `initial`?

Reply:
We should add `table.grow` support, but yeah, for now the size will be fixed.